### PR TITLE
Homework_3 pull

### PR DESCRIPTION
**Pull request by Vadoomz**

### CODE REVIEW

File: Homework_3.py
Comment (klymenok) [inline Homework_3.py]: There shouldn't be a transformation to `int`. The `d` variable must be a string

File: Homework_4.py
Comment (klymenok) [inline Homework_4.py:6]: Remove this code from the current PR and create a separate one. Please, take to account some comments below

File: Homework_4.py
Comment (klymenok) [inline Homework_4.py:9]: It is better to use here int() transformation since according to .isdigit() check the `enter` value cannot contain a dot

File: Homework_4.py
Comment (klymenok) [inline Homework_4.py:11]: It is better to use here int() transformation since according to .isdigit() check the enter value cannot contain a dot

**Review by klymenok — CHANGES_REQUESTED** [review]
Small changes are required for HW2. HW3 files should be removed from this PR

File: Homework_3.py
Comment (Vadoomz) [reply Homework_3.py]: Changed variable d to string

File: Homework_4.py
Comment (Vadoomz) [reply Homework_4.py:6]: What is the point to create another PR? Do I need to create another branch for it?

Comment (klymenok) [reply Homework_4.py:6]: No, you should make changes here and add as another commit

**Comment by klymenok** [comment]
You didn't fix your solution according to all comments

**Comment by klymenok** [comment]
And don't merge PR's before you get an approval

File: Homework_4.py
Comment (klymenok) [reply Homework_4.py:6]: You should create separate branch for every homework, and, as a result, there will be separates Pull Requests

**Pull request merged**
